app/lib/services/voip/CallLifecycle.test.ts

```
@@ -0,0 +1,488 @@
+		getLastVoipToken: jest.fn(() => ''),
+		stopNativeDDPClient: jest.fn(),
+		stopVoipCallService: jest.fn(),
+		addListener: jest.fn(),
+		removeListeners: jest.fn()
+	}
+}));
+jest.mock('../../../containers/ActionSheet', () => ({
+	hideActionSheetRef: jest.fn()
+}));
+
+import type { IClientMediaCall } from '@rocket.chat/media-signaling';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)